app/src/main/java/com/peacecorps/pcsa/support_services/SupportServices.java

```diff
@@ -0,0 +1,31 @@
+package com.peacecorps.pcsa.support_services;
+
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+
+import com.peacecorps.pcsa.R;
+
+public class SupportServices extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_support_services);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        FloatingActionButton fab = (FloatingActionButton) findViewById(R.id.fab);
+        fab.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Snackbar.make(view, "Replace with your own action", Snackbar.LENGTH_LONG)
```

Comment:
Like I have explained below, this is auto generated code. You'll see the necessary changes when I am working on the individual layouts, in this case it would be **Support Services**. Please ignore this for now